Updated Readme - added boundary as dependency. (#10)
am-kantox/estructura: README.md

```elixir
def deps do
  [
    {:estructura, "~> 0.1"},

>     {:boundary, "~> 0.9", runtime: false}

Reviewer: ```suggestion
    # optionally you might want to add `boundary` library 
    # it is used by `estructura` and many other projects
    # more info: https://hexdocs.pm/boundary
    {:boundary, "~> 0.9", runtime: false}
```

  ]
end
```

> Also, add `boundary` as dependency since that is used in present project.

Reviewer: ```suggestion
I suggest adding [`boundary`](https://hexdocs.pm/boundary) as a dependency since that is used in this project.
```
